### ISSUE TITLE
Branch is undefined

### ISSUE THREAD

**Issue by ckipp01** [amondnet/now-deployment#31]
Hello! Thanks for the great action. I've noticed that when I check in the Zeit Dashboard, the branch name is undefined that caused the deployment. Am I potentially missing something, or is this just not being set?

My setup:
```yaml
      - name: Zeit Now Deployment
        uses: amondnet/now-deployment@v2.0.0
        with:
          zeit-token: ${{ secrets.ZEIT_TOKEN }}
          now-org-id: ${{ secrets.ORG_ID }}
          now-project-id: ${{ secrets.PROJECT_ID }}
          now-args: '--prod'
```

Everything deploys correctly, but you can see in the deployments page that the commit message is there, but no branch name

<img width="507" alt="Screenshot 2020-02-23 at 13 41 27" src="https://user-images.githubusercontent.com/13974112/75112264-4beee880-5642-11ea-954d-9b5bfd67de1e.png">

Then when you click into the deployment, you'll again see the missing branch name that leads you to `tree/undefined` which will obviously 404 on github.

<img width="481" alt="Screenshot 2020-02-23 at 13 42 45" src="https://user-images.githubusercontent.com/13974112/75112286-a4be8100-5642-11ea-96ff-7a7ca3dadfa0.png">

**Comment by ckipp01** [comment]
Looking at it, my naive thought is that there is a missing key here:
https://github.com/amondnet/now-deployment/blob/1defb913612fc8d5369537fd690a3943dcceb1a9/index.js#L80-L104

However, I'm unable to quickly find it in the Zeit Deployment api docs.

**Comment by amondnet** [comment]
@ckipp01 
I think `githubCommitRef` needs to be added. 
Can you tell me if `branch` is displayed after setting like below?

```yaml
      - name: Zeit Now Deployment
        uses: amondnet/now-deployment@v2.0.0
        with:
          zeit-token: ${{ secrets.ZEIT_TOKEN }}
          now-org-id: ${{ secrets.ORG_ID }}
          now-project-id: ${{ secrets.PROJECT_ID }}
          now-args: '--prod -m githubCommitRef=YOUR_BRANCH' #-m githubCommitRef=YOUR_BRANCH
```

**Comment by ckipp01** [comment]
Just tested. 👍 the branch appears when that is filled in.

**Comment by amondnet** [comment]
@ckipp01 Thank you. I will fix it in the next release([v2.0.2](https://github.com/amondnet/now-deployment/milestone/6)).
